kanbananza-backend/src/main/controllers/column.js

```diff
@@ -67,10 +67,51 @@ const index = async (req, res, next) => {
       );
     }
 
-    res.status(200).json(column.toDTO());
+    return res.status(200).json(column.toDTO());
+  } catch (e) {
+    return next(e);
+  }
+};
+
+const update = async (req, res, next) => {
+  try {
+    let column;
+
+    column = await columnService.findColumnById(req.params.id);
+
+    if (column === null) {
+      return next(
+        new HttpError({
+          code: 404,
+          message: `Column with id ${req.params.id} does not exist.`,
+        })
+      );
+    }
+
+    const updatedInfo = {
+      name: req.body.name !== undefined ? req.body.name : column.name,
+      boardId:
+        req.body.boardId !== undefined ? req.body.boardId : column.boardId,
+      order: req.body.order !== undefined ? req.body.order : column.order,
+    };
```

Comment:
harmonize between put where all info is provided and patch where provided info is partial